peachjam/tasks.py

```diff
@@ -263,3 +263,12 @@ def generate_judgment_summary(doc_id):
         return
     log.info(f"Summarizing judgment {doc_id}")
     doc.generate_summary()
+
+
+@background(queue="peachjam", remove_existing_tasks=True)
+def send_timeline_emails():
```

Comment:
When does this get triggered? Daily?

As discussed, I think this should queue up separate tasks for each user that should be notified.